src/breadcrumbs/breadcrumbs.scss

```diff
@@ -21,45 +21,94 @@
 
 @mixin iui-breadcrumbs-item {
   display: flex;
+  align-items: center;
   line-height: $iui-component-height;
   height: $iui-component-height;
-  max-width: $iui-3xl * 2;
-  margin: 0 $iui-sm;
 
-  a {
+  // #region Component overrides
+  > a {
     @include iui-anchor;
+    padding: 0 $iui-s;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+
+    &:focus {
+      outline-offset: -1px;
+    }
   }
 
   .iui-button {
     @include iui-button-borderless;
-    margin: 0 (-$iui-s - 1);
+    border: none;
+
+    &-label {
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+    }
+  }
+
+  > * {
+    max-width: 26ch;
   }
 
-  > *,
-  .iui-button-label {
+  &:not([aria-current]) > * {
+    &,
+    &:hover,
+    &:active {
+      --_iui-button-foreground-color: var(--iui-color-foreground-primary);
+    }
+  }
+  // #endregion Component overrides
+}
+
+@mixin iui-breadcrumbs-text {
+  padding: 0 $iui-s;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+
+  @at-root a#{&} {
+    @include iui-anchor;
+
+    &:focus {
+      outline-offset: -1px;
+    }
+  }
+}
+
+@mixin iui-breadcrumbs-button {
+  @include iui-button;
+  @include iui-button-borderless;
+
+  border: none;
+  @include themed {
+    color: t(iui-color-foreground-primary);
+  }
+
+  &-label {
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
   }
 
-  &:not(.iui-current) > .iui-button {
+  @at-root [aria-current] & {
     @include themed {
-      color: t(iui-color-foreground-primary);
-    }
-
-    &:hover {
-      @include themed {
-        color: t(iui-color-foreground-primary-overlay);
-      }
+      color: t(iui-text-color);
     }
   }
 }
 
 @mixin iui-breadcrumbs-separator {
   display: flex;
+  margin: 0 $iui-xxs;
 
   svg {
     @include iui-icons-small;
```

Comment:
also this :DD

![image](https://user-images.githubusercontent.com/81580355/163560163-95258d20-7418-4b22-aa82-7a9488bb0eba.png)

Reply:
I don't think users will add classes on SVG components.

Reply:
If we really want to remove these info, positive, warning, negative styles, I could remove the `@include iui-icons-small;` and just place the sizing and color CSS by itself.

Reply:
> I could remove the @include iui-icons-small; and just place the sizing and color CSS by itself.

yes, please do that

Reply:
I have removed `@include iui-icons-small;` and only added what was needed.